web/src/main/java/org/mskcc/cbio/oncokb/controller/EvidenceController.java

```diff
@@ -222,6 +222,27 @@ ResponseEntity updateVUS(@ApiParam(value = "hugoSymbol", required = true) @PathV
         Set<ClinicalTrial> clinicalTrials = queryEvidence.getClinicalTrials();
         String propagation = queryEvidence.getPropagation();
         
+        List<String> cancerTypes = new ArrayList<>();
+        List<String> subTypes = new ArrayList<>();     
+        Boolean cancerEvidence = true;
+        if(cancerType == null) {
```

Comment:
EvidenceUtils.annotateEvidence(queryEvidence);
This is called at the begging, which will ensure empty string cancerType get converted to null